test/1.4/errors/T_ECAST.dml

```diff
@@ -45,6 +45,7 @@ method init() {
     cast(l, uint32);
     // no error!
     cast(a, uint32);
+    /// ERROR ECAST
```

Comment:
Also need error tests for casts between ptr and endian int

Reply:
No, cause they don't actually cause errors! The checks are misleading, I'll create another comment thread about that.